src/pages/moderators-page/components/main-content/components/review-card/index.tsx

```diff
@@ -148,7 +124,7 @@ const ReviewModeratorsCard: React.FC<ReviewModeratorsCardProps> = ({
 							Опубліковати
 						</Button>
 					)}
-					{(status.published || status.pending) && (
+					{(review.status === "approved" || review.status === "pending") && (
```

Comment:
I'd use it from enum

Reply:
how?

Reply:
As review.status === ReviewStatus.APPROVED

Reply:
but then it will be compared, for example "approved" with "Опубліковано", because in enam approved = "Опубліковано"
![image](https://github.com/user-attachments/assets/0478fd4a-f37b-4a52-a0a8-fe4e679e33f0)
The status from the backend "approved" is compared with the enam value "Опубліковано" and not with the key "approved"

Reply:
You understood me wrong. I meant enum like this enum ReviewStatus { APPROVED ='approved}. then it'd work. But ok, leave this,  it's not that important.